tests/flows/validator_set_sig.go

```diff
@@ -0,0 +1,235 @@
+package flows
+
+import (
+	"context"
+	"math/big"
+
+	runner_sdk "github.com/ava-labs/avalanche-network-runner/client"
+	"github.com/ava-labs/subnet-evm/accounts/abi/bind"
+	exampleerc20 "github.com/ava-labs/teleporter/abi-bindings/go/Mocks/ExampleERC20"
+	validatorsetsig "github.com/ava-labs/teleporter/abi-bindings/go/Utilities/ValidatorSetSig"
+	"github.com/ava-labs/teleporter/tests/interfaces"
+	"github.com/ava-labs/teleporter/tests/utils"
+	. "github.com/onsi/gomega"
+)
+
+func ValidatorSetSig(network interfaces.LocalNetwork) {
```

Comment:
doesn't have to be this PR, but we new `utils` folder for contracts, I'm thinking our e2e tests structure should follow that of the contracts. So would be `tests/flows/utilities/validator_set_sig.go` same for `teleporter_registry` tests, etc.

Reply:
and teleporter e2e tests would be in `tests/flows/teleporter`

Reply:
Yeah we can add it to #410